### PR TITLE
fix clickhouse vector store in hub

### PR DESCRIPTION
**Pull request by gingerwizard**
# Description
Fix name in llama hub

## Type of Change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Added new unit/integration tests
- [x] Added new notebook (that tests end-to-end)
- [x] I stared at the code and made sure it makes sense

# Suggested Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added Google Colab support for the newly added notebooks.
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I ran `make format; make lint` to appease the lint gods

### CODE REVIEW

File: llama-index-integrations/vector_stores/llama-index-vector-stores-clickhouse/pyproject.toml [hunk 1]
```diff
@@ -7,6 +7,11 @@ check-filenames = true
 check-hidden = true
 skip = "*.csv,*.html,*.json,*.jsonl,*.pdf,*.txt,*.ipynb"
 
+[tool.llamahub]
```
Comment (logan-markewich) [inline llama-index-integrations/vector_stores/llama-index-vector-stores-clickhouse/pyproject.toml:10]: Sorry lol small adjustment (still figuring out the structure we need for llamahub)

Lets do this

```
[tool.llamahub]
contains_example = false
import_path = "llama_index.vector_stores.clickhouse"

[tool.llamahub.class_authors]
ClickHouseVectorStore = "ClickHouse"
```

**Comment by logan-markewich** [comment]
@gingerwizard can you merge main one more time? There was some weird circular import issues in main (rip)

**Comment by gingerwizard** [comment]
@logan-markewich try now

**Comment by gingerwizard** [comment]
k this is good i think @logan-markewich

**Pull request merged**